models/datasets/README.md

```diff
@@ -0,0 +1,150 @@
+# Morpheus Datasets
+
+Small datasets for testing training scripts, inference scripts, and pipelines.
+
+## Anomalous Behavioral Profiling (ABP)
+
+This is a labeled dataset of 1241 nv-smi logs generated once per minute from a single Tesla V100 in our lab environment running either GPU malware or benign workflows.
+
+### Sample Training Data
+
+- [abp-sample-nvsmi-training-data.json](./training-data/abp-sample-nvsmi-training-data.json)
+
+### Pipeline Validation Data
+The same data in both csv and jsonlines
+
+- [abp-validation-data.csv](./validation-data/abp-validation-data.csv)
+- [abp-validation-data.jsonlines](./validation-data/abp-validation-data.jsonlines)
+
+
+## Digital Fingerprinting (DFP) Data
+
+### DFP Azure Logs
```

Comment:
We should avoid empty headers like this. Use this generic DFP Azure Logs section to talk about the logs, then differentiate in the sub-sub headings below between sample training data, pipeline validation, etc.

Reply:
fixed